www/netj00/HW1/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Titulní Strana Unicornu</title>
+    <link rel="stylesheet" type="text/css">
+</head>
+<body>
+<section>
+    <span> Unicorn a.s</span>
+    <nav>
+        <ul>
+            <li><a href="index.html">Galerie</a></li>
+            <li><a href="index.html">Kontakt</a></li>
+            <li><a href="index.html">O stránkách</a></li>
+        </ul>
+    </nav>
+    <article>
+        <h1>Titulní strana</h1>
+        <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Cras pede libero, dapibus nec, pretium sit amet,
+            tempor quis. Fusce aliquam vestibulum ipsum. Vestibulum fermentum tortor id mi. Duis ante orci, molestie
+            vitae vehicula venenatis, tincidunt ac pede. Praesent dapibus. Integer in sapien. Neque porro quisquam est,
+            qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora
+            incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Vestibulum fermentum tortor id mi. Duis ante
+            orci, molestie vitae vehicula venenatis, tincidunt ac pede. Aliquam ornare wisi eu metus. Aliquam erat
+            volutpat. Sed elit dui, pellentesque a, faucibus vel, interdum nec, diam. Morbi imperdiet, mauris ac auctor
+            dictum, nisl ligula egestas nulla, et sollicitudin sem purus in lacus. Morbi leo mi, nonummy eget tristique
+            non, rhoncus non leo. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+            ea commodo consequat. Duis bibendum, lectus ut viverra rhoncus, dolor nunc faucibus libero, eget facilisis
+            enim ipsum id lacus.
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Etiam posuere lacus quis dolor.
+            Etiam neque. Etiam quis quam. Etiam egestas wisi a erat. Nullam sit amet magna in magna gravida vehicula.
+            Nulla turpis magna, cursus sit amet, suscipit a, interdum id, felis. Fusce tellus. Fusce tellus odio,
+            dapibus id fermentum quis, suscipit id erat. Proin mattis lacinia justo.
+            Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.
+            Aenean fermentum risus id tortor. Maecenas sollicitudin. Sed vel lectus. Donec odio tempus molestie,
+            porttitor ut, iaculis quis, sem. Aliquam erat volutpat. Vivamus luctus egestas leo. Cum sociis natoque
+            penatibus et magnis dis parturient montes, nascetur ridiculus mus. Nulla pulvinar eleifend sem. Aliquam
+            erat volutpat. Duis risus. Aliquam ornare wisi eu metus. Class aptent taciti sociosqu ad litora torquent
+            per conubia nostra, per inceptos hymenaeos. Nullam lectus justo, vulputate eget mollis sed, tempor sed
+            magna. Etiam dui sem, fermentum vitae, sagittis id, malesuada in, quam.
+        </p>
+        <ul>
```

Comment:
v &lt;**ul**&gt; musí být &lt;**li**&gt;, je to tedy špatné použití &lt;**ul**&gt; každý produkt by měl být jedno &lt;**li**&gt;